statsmodels/base/tests/test_generic_methods.py

```diff
@@ -227,28 +400,50 @@ def setup(self):
         x = self.exog
         np.random.seed(987689)
         y_count = np.random.poisson(np.exp(x.sum(1) - x.mean()))
-        model = sm.Poisson(y_count, x)  #, exposure=np.ones(nobs), offset=np.zeros(nobs)) #bug with default
+        model = sm.Poisson(y_count, x)
+        # use start_params to converge faster
+        start_params = np.array([0.75334818, 0.99425553, 1.00494724, 1.00247112])
+        self.results = model.fit(start_params=start_params, method='bfgs',
+                                 disp=0)
+
+
+class TestGenericPoissonOffset(CheckGenericMixin):
+
+    def setup(self):
+        #fit for each test, because results will be changed by test
+        x = self.exog
+        nobs = x.shape[0]
+        np.random.seed(987689)
+        y_count = np.random.poisson(np.exp(x.sum(1) - x.mean()))
+        model = sm.Poisson(y_count, x, offset=0.01 * np.ones(nobs),
+                           exposure=np.ones(nobs)) #bug with default
         # use start_params to converge faster
         start_params = np.array([0.75334818, 0.99425553, 1.00494724, 1.00247112])
         self.results = model.fit(start_params=start_params, method='bfgs',
                                  disp=0)
 
-        #TODO: temporary, fixed in master
+        self.predict_kwds_5 = dict(exposure=0.01 * np.ones(5), offset=np.ones(5))
         self.predict_kwds = dict(exposure=1, offset=0)
 
+
 class TestGenericNegativeBinomial(CheckGenericMixin):
 
     def setup(self):
         #fit for each test, because results will be changed by test
         np.random.seed(987689)
         data = sm.datasets.randhie.load()
         exog = sm.add_constant(data.exog, prepend=False)
-        mod = sm.NegativeBinomial(data.endog, data.exog)
-        start_params = np.array([-0.0565406 , -0.21213599,  0.08783076,
+        #mod = sm.NegativeBinomial(data.endog, data.exog)
+        mod = sm.NegativeBinomial(data.endog, exog)
+        start_params = np.array([0.1, -0.0565406 , -0.21213599,  0.08783076,
                                  -0.02991835,  0.22901974,  0.0621026,
                                   0.06799283,  0.08406688,  0.18530969,
                                   1.36645452])
+        start_params = np.array([-0.05783623, -0.26655806,  0.04109148, -0.03815837,
```

Comment:
This might be the diff displaying things weirdly, but it looks like `start_params` is defined and then defined again three lines later.